### PR TITLE
DX-1764 EDP 1.12.0 Release notes

### PR DESCRIPTION
**Pull request by tbuchaillot**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---
https://tyktech.atlassian.net/browse/DX-1764
https://tyktech.atlassian.net/browse/TT-13408

### Preview Link

https://deploy-preview-5698--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0/
<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Enterprise Developer Portal v1.12.0, detailing new features, enhancements, and fixes.
- Introduced embedded Tyk Identity Broker, allowing SSO without a separate deployment.
- Enhanced admin capabilities for creating apps and credentials directly from the portal UI.
- Implemented stronger password policies configurable via the portal admin UI.
- Added credential expiration notifications to improve security and user management.
- Upgraded to Golang 1.22 for better performance and security.
- Fixed several bugs, including issues with dropdown custom attributes and certificate uploads in Kubernetes.
- Addressed high-priority security vulnerabilities with fixes for multiple CVEs.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>portal-1.12.0.md</strong><dd><code>Add release notes for Tyk Enterprise Developer Portal v1.12.0</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md

<li>Added release notes for Tyk Enterprise Developer Portal v1.12.0.<br> <li> Documented new features such as embedded Tyk Identity Broker and <br>app/credential creation.<br> <li> Included details on security fixes and bug fixes.<br> <li> Provided upgrade instructions and download links.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5698/files#diff-d96d884f7e9e1d3d66424d600f429204a72067a361c3cb8f9436ca1d145dcb94">+235/-0</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/assets/others/enterprise-developer-portal-swagger.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ tags:
   - name: Tutorials for API Products
     description: Tutorials that are defined for the API products
   - name: API documentation for API Products
-    description: OpenAPI specs for APIs included into the API Products
+    description: OpenAPI specs for APIs included into the API Prodcuts
```
Comment (sharadregoti) [inline tyk-docs/assets/others/enterprise-developer-portal-swagger.yaml]:
```suggestion
    description: OpenAPI specs for APIs included into the API Products
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+Fixed the following high priority CVEs identified in the Tyk Gateway, providing increased protection against security
+vulnerabilities:
+
+- [CVE-2024-34158](https://nvd.nist.gov/vuln/detail/CVE-2024-34158)
+- [CVE-2024-34156](https://nvd.nist.gov/vuln/detail/CVE-2024-34156)
+- [CVE-2022-30635](https://nvd.nist.gov/vuln/detail/CVE-2022-30635)
+
+</details>
+</li>
+</ul>
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md]: @alienor This has to be removed right?

**Review by sharadregoti — CHANGES_REQUESTED** [review]

File: tyk-docs/data/page_available_since.json [hunk 3]
```diff
@@ -6029,6 +6029,11 @@
             "/docs/5.5/": "/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.11.0/",
             "/docs/nightly/": "/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.11.0/"
         },
+        "/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0/": {
+            "/docs/": "/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0/",
+            "/docs/5.5/": "/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0/",
+            "/docs/nightly/": "/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0/"
+        },
```
Comment (sharadregoti) [inline tyk-docs/data/page_available_since.json]:
```suggestion
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md (hunk 2 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md]: @sharadregoti let me change Tyk Gateway mention for EDP here- Why we would want to remove this section?

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->
+<ul>
+<li>
+<details>
+<summary>Embedded Tyk Identity Broker</summary>
+
+From this release, you can configure the portal to serve an internal Tyk Identity Broker. This means that you don't need to deploy a separate Tyk Identity Broker service to SSO into the portal.
+This enables a new section under the portal admin UI where admins can manage SSO profiles for admins and developers.
+{{< img src="/img/dashboard/portal-management/enterprise-portal/1.12.0-embedded-tib.png" width=500px alt="SSO profiles">}}
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md:73]: I cannot read this text 
image

Comment (letzya) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.12.0.md:73]: I love the use of images! is so helpful. However, I struggle reading the content @sharadregoti . If we can improve it with a separate iteration, we can keep this. 

image

.

The diff between the content font size and the  font size in the images is big

**Review by sharadregoti — APPROVED** [review]

**Comment by sharadregoti** [comment]
TODO: Change the release date when merged.

**Pull request merged**